src/file-chooser.c

```diff
@@ -564,6 +566,40 @@ save_file_done (GObject *source,
   g_task_run_in_thread (task, send_response_in_thread_func);
 }
 
+/* Calling Lookup on a nonexisting path does not work, so we
+ * pull the doc id out of the path manually.
+ */
+static gboolean
+looks_like_document_portal_path (const char *path, char **guessed_docid)
```

Comment:
The coding style is inconsistent, but it would be great if we could bring it back to something approaching the usual GLib style:

```suggestion
looks_like_document_portal_path (const char *path,
                                 char **guessed_docid)
```
```